interfaces/builtin/bluetooth_control.go

```diff
@@ -0,0 +1,59 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const bluetoothControlConnectedPlugAppArmor = `
+# Description: Allow managing the kernel side Bluetooth stack. Reserved
+#  because this gives privileged access to the system.
+# Usage: reserved
+
+  network bluetooth,
+  # For crypto functionality the kernel offers
+  network alg,
+
+  capability net_admin,
+
+  # File accesses
+  /sys/bus/usb/drivers/btusb/     r,
+  /sys/bus/usb/drivers/btusb/**   r,
+  /sys/class/bluetooth/           r,
+  /sys/devices/**/bluetooth/      rw,
+  /sys/devices/**/bluetooth/**    rw,
+`
+
+const bluetoothControlConnectedPlugSecComp = `
+# Description: Allow managing the kernel side Bluetooth stack. Reserved
+#  because this gives privileged access to the system.
+# Usage: reserved
+bind
```

Comment:
@morphis - you misunderstood. I agree that bluetooth-control should work without network. I was saying I'm surprised it does work without network since you only have the one additional seccomp call for this interface, and was trying to make sure you didn't test with the network interface connected.

Please verify that network is not connected and re-run your tests on at least x86-64, adding any seccomp calls here as needed. Due note we allow a few things in the default seccomp template, like 'socket' and 'connect' for other reasons. If bind is truly all you need, that's fine and thanks for confirming.